theme_kea/views/snippets/s_cover.xml

```diff
@@ -3,23 +3,32 @@
     <template id="s_cover" inherit_id="website.s_cover">
         <!-- Section -->
         <xpath expr="//section" position="attributes">
-            <attribute name="class" add="o_full_screen_height" separator=" "/>
-            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/07_002","flip":[]}</attribute>
+            <attribute name="class" add="s_parallax_no_overflow_hidden oe_img_bg o_full_screen_height" remove="parallax s_parallax_is_fixed bg-black-50 remove" separator=" "/>
```

Comment:
Why do you remove `.remove`? 😄 
I think you are trying to remove the parallax effect here. So why do you add `.s_parallax_no_overflow_hidden`?